UserList-LIUCHEN/UserList-LIUCHEN/Utility/UIImageView+loadImage.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  UIImageView+loadImage.swift
+//  UserList-LIUCHEN
+//
+//  Created by LC on 2020/6/23.
+//  Copyright © 2020 LC. All rights reserved.
+//
+
+import UIKit
+
+/// Can use Kingfisher instead
+extension UIImageView {
+
+    
+    /// Download and Cache Image
+    /// - Parameter urlString: Image url
+    /// - Returns: URLSessionDownloadTask
+    func loadImage(urlString: String) -> URLSessionDownloadTask{
+        let session = URLSession.shared
+        let imageCache = NSCache<AnyObject, UIImage>()
```

Comment:
The cache object here gets created within the function scope and also released within the function scope, so there actually won't be cache at all because it gets destroyed at the end of the function and recreated every time the function is called.

Or did I miss something?

Reply:
It’s an mistake should be put it here

let imageCache = NSCache<AnyObject, UIImage>()
extension UIImageView {
 ......
   func loadImage(){}
}